docs/swagger.yml

```diff
@@ -519,6 +519,11 @@ definitions:
         default: 30
         format: int32
         description: Hot functions idle timeout before termination. Value in Seconds
+      metadata:
+        type: object
```

Comment:
i suppose this should be a `string` but not the `object`. With `string` type you can limit the size of the object that would be passed to the generated client. https://swagger.io/docs/specification/data-models/data-types/#string

Reply:
I want to support JSON natively on the API  and object works pretty nice with most rendered swagger clients (`map[string]interface{}`) in go, `Map<String,Object>` in java etc. 

The length constraints apply to individual keys and values  so having this as a string and forcing clients to re-encode/Decode to JSON seemed unnecessarily painful.

Reply:
(I don't see a strong reason to change this)

Reply:
object seems fine to me, too